docs/streams/developer-guide/dsl-api.html

```diff
@@ -956,15 +956,15 @@ <h4 class="anchor-heading"><a id="streams_concepts_globalktable" class="anchor-l
 KTable&lt;byte[], Long&gt; aggregatedStream = groupedStream.aggregate(
     () -&gt; 0L, /* initializer */
     (aggKey, newValue, aggValue) -&gt; aggValue + newValue.length(), /* adder */
-    Materialized.as(&quot;aggregated-stream-store&quot;) /* state store name */
+    Materialized.<String, Long, KeyValueStore<Bytes, byte[]>>as(&quot;aggregated-stream-store&quot;) /* state store name */
```

Comment:
Please escape the `<` and `>` with `&lt;` and `&gt;`. You can check the above `KTable&lt;byte[], Long&gt; aggregatedStream` example => `KTable<byte[], Long> aggregatedStream`

Reply:
@showuon thanks a lot! Fixed it.